src/Views/General.vala

```diff
@@ -118,6 +118,20 @@ public class MouseTouchpad.GeneralView : Gtk.Grid {
         pointer_speed_help.xalign = 0;
         pointer_speed_help.get_style_context ().add_class (Gtk.STYLE_CLASS_DIM_LABEL);
 
+        var double_click_speed_adjustment = new Gtk.Adjustment (400, 300, 1500, 0.1, 0.1, 0.1);
+
+        var double_click_speed_scale = new Gtk.Scale (Gtk.Orientation.HORIZONTAL, double_click_speed_adjustment);
+        double_click_speed_scale.draw_value = false;
+        double_click_speed_scale.add_mark (400, Gtk.PositionType.TOP, null);
+        double_click_speed_scale.width_request = 250;
+
+        var double_click_speed_help = new Gtk.Label (_("This adjusts double click speed of the pointer"));
```

Comment:
I wonder if something more explanatory like this makes sense:

```suggestion
        var double_click_speed_help = new Gtk.Label (_("How quickly two clicks in a row will be treated as a double click for both a mouse and touchpad."));
```

Reply:
What about `Sets how quickly two clicks in a row will be treated as a double click for both the mouse and touchpad.` ? Starting with 'how ' feels more like a question.

Reply:
I think that's pretty normal in English to start a sentence with "How" as an explanation and not necessarily as a question.

"How quickly time flies" vs "How quickly does time fly?".

Reply:
`How quickly time flies` is an idiom, so it might now follow the rules. Most of the time starting the sentence with How usually denotes _exclamation_ , which the above sentence looks like . I am no grammar expert but this might just be one of those linguistic differences between countries.

Reply:
Also **Sets** is an Action Verb, without it sentence just doesn't look complete.

Reply:
The scale is the setting, not the action, though. Above, we don't say, "Turning this on will make 
it so that pressing the control key will highlight the position of the pointer," we say, "Pressing the control key will highlight the position of the pointer." The help text explains the setting.

You can also see this setting as "how fast the car goes" or "how much sugar to put into the cookie." Starting the explanatory text with "how" feels very natural and in line with our usual elementary copy style.

Reply:
If we wanted to be more proper but more stilted, we could say something like, "The speed at which two clicks will be treated as a double click…" but that's much less natural/conversational to me than "How quickly…"

Reply:
Since this is a help message along with the scale it implies that the action is the scale change and label is the action performed and if similar thing is used somewhere else, you can go with it. However I am not entirely convinced here.

Reply:
> The speed at which two clicks will be treated as a double click

This looks much better. Might want to take input on how will both be translated in other languages.

Reply:
Gnome uses `Double-Click Delay` for the label here instead of `Double Click speed` might add more context here. I can then go with sentence starting with `How`.